packages/next-server/server/load-components.ts

```diff
@@ -8,14 +8,19 @@ function interopDefault(mod: any) {
 
 export async function loadComponents(distDir: string, buildId: string, pathname: string) {
   const documentPath = join(distDir, SERVER_DIRECTORY, CLIENT_STATIC_FILES_PATH, buildId, 'pages', '_document')
-  const appPath = join(distDir, SERVER_DIRECTORY, CLIENT_STATIC_FILES_PATH, buildId, 'pages', '_app')
-  const [buildManifest, reactLoadableManifest, Component, Document, App] = await Promise.all([
+  const [buildManifest, reactLoadableManifest, Component, Document] = await Promise.all([
     require(join(distDir, BUILD_MANIFEST)),
     require(join(distDir, REACT_LOADABLE_MANIFEST)),
     interopDefault(requirePage(pathname, distDir)),
     interopDefault(require(documentPath)),
-    interopDefault(require(appPath)),
   ])
 
+  let App
+  try {
+    App = interopDefault(require('private-next-pages/_app'))
```

Comment:
Typescript doesn't like when I import the constant from `next`

Reply:
Note that load-components is not transpiled, so it has to require from `.next` etc.